src/main/java/com/uid2/admin/store/writer/ScopedStoreWriter.java

```diff
@@ -59,7 +59,11 @@ public void upload(String data, JsonObject extraMeta) throws Exception {
         fileManager.uploadFile(location, dataFile, data);
         fileManager.uploadMetadata(metadata.getJson(), dataType, scope.getMetadataPath());
 
-        provider.loadContent(provider.getMetadata());
+        //The loadContent method is trying to read and parse this data as if it were unencrypted
+        // which is causing the deserialization error for encrypted json
+
+       // provider.loadContent(provider.getMetadata());
+
```

Comment:
But commenting this line out would fail a certain tests
<img width="331" alt="Screenshot 2024-06-30 at 2 59 45 AM" src="https://github.com/IABTechLab/uid2-admin/assets/104238560/8adafcf1-adc1-4043-aac1-df3fc3e9aff8">